bin/ch/ch.cpp

```diff
@@ -671,6 +671,10 @@ HRESULT ExecuteTest(const char* fileName)
         }
     }
 Error:
+    if( fileContents != nullptr)
+    {
+        free((char*)fileContents);
```

Comment:
That's true for the old API. The new API takes ArrayBuffer: https://github.com/Microsoft/ChakraCore/blob/master/lib/Jsrt/Jsrt.cpp#L4491